fusor-ember-cli/app/components/new-environment-modal.js

```diff
@@ -4,7 +4,7 @@ export default Ember.Component.extend({
 
   envLabelName: Ember.computed('name', function() {
     if (this.get('name')) {
-      var label = this.get('name').underscore();
+      var label = this.get('name').trim();
       return label.replace(/[^A-Z0-9]/ig, "_");
```

Comment:
the function `underscore()` actually also down-cased which was strange. 
the regex replace is similar to `underscore()` but without down-casing.